package.json

```diff
@@ -4,9 +4,10 @@
     "codegen": "graph codegen subgraph.yaml",
     "build": "graph build subgraph.yaml",
     "build-ipfs": "graph build --ipfs /ip4/127.0.0.1/tcp/5001 subgraph.yaml",
-    "build-wast": "graph build -t wast subgraph.yaml"
+    "build-wast": "graph build -t wast subgraph.yaml",
+    "deploy": "graph deploy subgraph.yaml --watch --verbosity debug --node http://35.242.224.197:8020/ --ipfs /ip4/35.242.224.197/tcp/8030 --subgraph-name adchain"
```

Comment:
Can we make the IP address for all these `127.0.0.1` to match the `build-ipfs` command above?

Reply:
yep